app/src/main/java/com/example/recipeopedia/activities/FavoriteRecipeDetailsActivity.java

```diff
@@ -90,6 +90,8 @@ public void onClick(View v) {
                 favoriteRecipe.put(KEY_ATTEMPT_PICTURE, new ParseFile(photoFile));
                 favoriteRecipe.saveInBackground();
                 Toast.makeText(FavoriteRecipeDetailsActivity.this, R.string.picture_uploaded, Toast.LENGTH_SHORT).show();
+                goMainActivity();
+                // TODO: navigate to FavoriteRecipesFragment ?
```

Comment:
If you want to do this, you'd add some custom key-value pair to the Intent when you start the activity, then unparcel it in onCreate and use that information to load the fragment you want ([example](https://stackoverflow.com/a/3913720))